server/neural_network_interface.py

```diff
@@ -32,10 +32,11 @@ def generate_move(self, sid, gesture_data, message_data):
                 movement_ct)
 
 class NeuralNetworkBlocksworldModel(BlocksworldModel):
-    def __init__(self, h5_paths):
+    def __init__(self, h5_paths, ambiguity_threshold=0.3):
         (self.flip_model, self.colors_model, self.letters_model) = runner.load_models(h5_paths)
         self.tokenizer = core.build_tokenizer(core.load_vocabulary())
-        BlocksworldModel.__init__(self)
+        super().__init__()
```

Comment:
This line is causing a build error (the previous way is the way `pylint` accepts). Also, isn't this traditionally at the top or the bottom?